tests/expected/bounded-arbitrary/reverse_vec/vec.expected

```diff
@@ -0,0 +1,10 @@
+Checking harness check_reverse_is_its_own_inverse_should_fail...\
+
+Failed Checks: unwinding assertion loop 0 \
```

Comment:
Which harness has a failed unwinding assertion? We should increase the unwind value for it and make sure the check we're expecting to fail is the one failing.